### DISCUSSION TITLE
Table Panel Auto-Scroll Option

### DISCUSSION THREAD

**Discussion by mvtechteam** [grafana/grafana#64897] [Dashboards, Panels & Visualizations]
It would be nice to have an option within the table panel where you can enable auto-scrolling of the panel when it is presented on the dashboard. I know that there is pagination, but when you have a table on a dashboard used for monitoring (where you are not interacting with the dashboard), it would be cool if the table would infinitely auto-scroll through the list so you could periodically see all the results in the table on a monitoring dashboard. I looked into the Dynamic Text plugin, but I love the simplicity of the table panel.
Below is a sketch of how the additional option for auto-scrolling in the table panel could look.

New Options:
- **Enable Auto-Scrolling** = Toggle the auto-scroll feature (which would show or hide the additional auto-scroll options)
- **Auto-Scrolling Behavior** = Sets the type of auto-scroll that should occur. **Bounce** would be where it scrolls down to the bottom, then scrolls going back up and so on, while **Reset to Top** would be where it scrolls to the bottom, then snaps back to the top and continues scrolling down.
- **Auto-Scrolling Speed** = an interval number that would set the speed of the scroll. Since the table height and number of entries is dynamic, this should be an independent rate of speed regardless of the height. The lower the number, the slower it gets.

![table autoscroll](https://user-images.githubusercontent.com/116840530/225620783-72e06c62-74ab-4d55-b7b7-6dec8c396a29.png)

**Comment by usmangt** [comment]
Hi @mvtechteam,

By chance, I stumbled on this [post](https://community.grafana.com/t/how-to-autoscroll-a-table-in-the-dashboard/81264) and found out that it is possible. Please check the community post for more details.

**Reply by mvtechteam** [reply]
Thank you for sharing this. Although, the post you provided is a code insert for the Dynamic Text plugin. This feature request is something I would like to see for the native Table panel within Grafana, as I like the native table with all of its easy-to-setup features. Hopefully, the devs will see this discussion :)

**Comment by nnn149** [comment]
I also need this!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82554. Feel free to continue the discussion around this there. Thank you!
